app/uk/gov/hmrc/agentsubscriptionfrontend/config/frontendAppConfig.scala

```diff
@@ -69,4 +72,8 @@ class FrontendAppConfig extends AppConfig with StrictConfig with ServicesConfig
     PostcodesLoader.load("/po_box_postcodes_abp_49.csv").map(x => x.toUpperCase.replace(" ", "")).toSet
   override lazy val journeyName: String = getConfString("address-lookup-frontend.journeyName", "")
   override lazy val agentServicesAccountUrl: String = s"$servicesAccountUrl/$servicesAccountPath"
+
+  override lazy val domainWhiteList =
+    runModeConfiguration.getStringList("continueUrl.domainWhiteList").getOrElse(Collections.emptyList()).toSet
```

Comment:
The domainWhitelist expects ```Set[String]```. So it doesn't make much difference to use Seq instead of List.